__tests__/integration.test.js

```diff
@@ -295,96 +296,132 @@ describe("POST", () => {
+        });
+    });
 
-      test("responds with 400 if passed username is missing", () => {
-        const newComment = {
-            body: "cool new comment!"
-          };
-        return request(app)
-          .post("/api/articles/2/comments")
-          .send(newComment)
-          .expect(400)
-          .then((request) => {
-            expect(request.body.msg).toEqual("Bad request!");
-          });
-      });
+    test("responds with 400 if passed body is missing", () => {
+      const newComment = {
+        username: "icellusedkars",
+      };
+      return request(app)
+        .post("/api/articles/2/comments")
+        .send(newComment)
+        .expect(400)
+        .then((request) => {
+          expect(request.body.msg).toEqual("Bad request!");
+        });
+    });
+
+    test("responds with 400 if passed username is missing", () => {
+      const newComment = {
+        body: "cool new comment!",
+      };
+      return request(app)
+        .post("/api/articles/2/comments")
+        .send(newComment)
+        .expect(400)
+        .then((request) => {
+          expect(request.body.msg).toEqual("Bad request!");
+        });
+    });
+
+    test("responds with 400 if passed username doesn't exist", () => {
+      const newComment = {
+        username: "jakerbp",
+        body: "cool new comment",
+      };
+      return request(app)
+        .post("/api/articles/2/comments")
+        .send(newComment)
+        .expect(400)
+        .then((request) => {
+          expect(request.body.msg).toEqual("Bad request!");
+        });
+    });
+  });
+});
 
-      test("responds with 400 if passed username doesn't exist", () => {
-        const newComment = {
-            username: "jakerbp",
-            body: "cool new comment",
-          };
+describe("DELETE", () => {
+  describe("/api/comments/:comment_id", () => {
+    test("responds with 204 upon success", () => {
+      return request(app).delete("/api/comments/1").expect(204);
+    });
+
+    test("responds with 404 if comment_id doesn't exist", () => {
+      return request(app)
+        .delete("/api/comments/9999")
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe('Comment with id 9999 does not exist!')
+        });
+    });
+
+    test("responds with 400 if comment_id is invalid", () => {
+      return request(app)
+        .delete("/api/comments/banana")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe('Bad request!')
+        });
+    });
+
+    test("responds with 404 if comment_id is missing", () => {
         return request(app)
-          .post("/api/articles/2/comments")
-          .send(newComment)
-          .expect(400)
-          .then((request) => {
-            expect(request.body.msg).toEqual('Bad request!');
+          .delete("/api/comments/")
+          .expect(404)
+          .then(({ body }) => {
+            expect(body.msg).toBe('Not found!')
```

Comment:
This last test isn't really necessary if you've already done a bad path test elsewhere in the project